src/EnergyPlus/SimAirServingZones.hh

```diff
@@ -153,7 +154,11 @@ namespace SimAirServingZones {
                              bool FirstHVACIteration,     // TRUE if first full HVAC iteration in an HVAC timestep
                              int AirLoopNum,              // Primary air loop number
                              int &CompIndex,              // numeric pointer for CompType/CompName -- passed back from other routines
-                             HVACSystemData *CompPointer);
+                             HVACSystemData *CompPointer, // equipment actual pointer
+                             int const &airLoopNum,       // index to AirloopHVAC
+                             int const &branchNum,        // index to AirloopHVAC branch
+                             int const &compNum           // index to AirloopHVAC branch component
+    );
```

Comment:
Adds three new arguments to SimAirLoopComponent() function that allows getting the pointer to the component simulated.